miso-web/src/main/webapp/scripts/sample_ajax.js

```diff
@@ -1384,32 +1389,60 @@ Sample.ui = {
     }
   },
   /**
-   * Returns an array of sample classes that correspond to given sample class IDs (for child samples)
+   * Returns an array of sample classes that correspond to given sample class IDs (for descendent samples)
+   * There must exist a path (of the same length) between all the starting sample classes and the targets.
    */
-  getChildSampleClasses: function (sampleClassIDs) {
-    return Sample.sampleClasses.filter(function (sc) {
-      return sampleClassIDs.indexOf(sc.id) != -1;
+  getMatchedDescentSampleClasses: function (sampleClassIDs) {
```

Comment:
This results in being able to choose to propagate from tissue to aliquot, but after entering aliquot details, and hitting save, it fails with the message "Parent Primary Tumor Tissue not permitted to have a child of type gDNA (aliquot)." I'm not sure if propagating from Tissue to Aliquot should be allowed, but my guess is no.